doc/HttpLuaModule.wiki

```diff
@@ -5384,7 +5384,7 @@ Inserts the specified (numerical or string) <code>value</code> at the head of th
 
 If <code>key</code> does not exist, it is created as an empty list before performing the push operation. When the <code>key</code> already takes a value that is not a list, it will return <code>nil</code> and <code>"value not a list"</code>.
 
-It never overrides the (least recently used) unexpired items in the store when running out of storage in the shared memory zone. In this case, it will immediately return <code>nil</code> and the string "no memory".
+It never overrides the (least recently used) unexpired items in the store when running out of storage in the shared memory zone. In this case, it will immediately return <code>false</code> and the string "no memory".
```

Comment:
right, I just use if rc == nil before I find the return value is wrong, even thought, the document should be modified. I just modify the document